external/vcm/tests/_coarsen_restarts_regression_tests/reference/area-weighted-model-level-without-agrid-winds-fv_core.res.json

```diff
@@ -0,0 +1 @@
4967, -28.03878315055515, 591.2685921140637], [-59.14215316856778, 670.0516389386883, -163.91002124874134]], [[771.8154131031828, 270.7323294164159, 261.4358151642227], [-112.46960077434454, -406.8367797713327, -779.2615497349803]], [[-139.14613335762664, -694.4265809650248, -13.79320726285438], [-766.7675308828857, -907.1825552799661, -154.60158641951134]]]], [[[[424.4019151778729, -455.8253068517292, -57.483981115045786], [-391.2668435567328, -478.2219541898708, -45.58999632365941]], [[-291.93305243797096, 898.1657070694149, 108.10875518493368], [-450.40061117401837, -639.3787153404606, 847.7047749031019]], [[-564.0567709206006, -307.053237155717, -524.448064361864], [175.95853370470266, 509.59357218259544, 498.0716793374409]], [[-280.7619437494215, -103.17079352460819, -340.0405439135245], [-452.5381503194711, 435.3962384399832, 67.97832894078094]], [[285.4727021901571, -329.6056587288071, -225.29329748315106], [-373.15928853539714, 191.98597422963144, 445.85105637018256]], [[-97.83287886674863, -228.79544366974747, -603.6197192324214], [-888.3878129729368, 297.29976490215665, -1.867100422687443]], [[-979.2494185402155, 370.7519310740756, 324.2945522892798], [99.27370977763756, -138.8857074823829, 539.0114891916322]]]], [[[[534.9948874379876, 903.0955157653805, 519.3089876461489], [253.34840296218806, 24.14597092266535, 865.5120357058371]], [[-594.1453977081394, -505.0423681928291, 626.1150506022924], [723.8807657648564, 153.71220446258252, 200.19210139605042]], [[-353.8635090528881, -87.33909794536905, -49.65237028127306], [-193.96667695121576, 645.8284354077687, -15.920005134783258]], [[230.73577766359435, 336.66868090638053, 37.95127819446877], [-27.113914930282323, -443.831475872396, 521.9175200366093]], [[10.216995364761246, 635.6785064679788, -551.7756950558701], [-419.9194891442152, -636.0835173202973, 824.6031201175159]], [[-175.99505510761955, -534.8269507621551, 67.40147707494769], [38.82355588656718, 155.47861094659277, 279.24037126640496]], [[7.590980745026847, -326.9134175417396, -787.5903055399301], [259.4353440229235, -419.18895740851406, 327.4215759870131]]]], [[[[-518.2347143407021, -852.6099446047097, 217.27979359633045], [-638.9506817572459, 310.52501752565024, -86.87470015188599]], [[625.4154837879888, -30.914293628859713, -249.04896257443045], [179.5868676917671, -116.45402282745297, -332.55953810169524]], [[-872.9038236934767, -311.78154433911305, 97.36243151995416], [167.53132605143867, -34.310124459537874, -416.3241932386889]], [[801.109653938307, -175.32898970790814, 653.031268944695], [-23.441072483616132, -299.5833146847884, 769.7072133118921]], [[-705.0847355777385, -43.97519048972128, -788.824182733945], [494.2930650223648, -148.62445096638015, 476.3101435422064]], [[-271.87365602550966, -236.98392628353164, 570.6687420863033], [-427.55998462223766, -312.5279107546991, -374.9999977807396]], [[406.1765810580566, 262.41067115299006, -643.3988970397526], [87.3452679958637, -198.3149368084531, 530.3419378585622]]]]]}}}
```

Comment:
no change necessary, but if we wanted to make these even smaller, could use lower precision for the floats. Especially given I imagine we're using a relatively large tolerance when doing the test.

Reply:
Interesting thought.  Since JSON is not a binary format there's not a super clean way of doing this, but if I truncate to five decimal points (e.g. through `xarray.Dataset.round`), the total size of the reference files is only reduced from 604K to 508K (fv3net itself is about 359M).  Therefore I've kept things as is.